apps/src/code-studio/utils.js

```diff
@@ -1,22 +1,34 @@
+import queryString from 'query-string';
+
 module.exports = {
   /**
    * Gets the URL querystring params.
    * @param name {string=} Optionally pull a specific param.
    * @return {object|string} Hash of params, or param string if `name` is specified.
    */
-  queryParams: function (name) {
-    var pairs = location.search.substr(1).split('&');
-    var params = {};
-    pairs.forEach(function (pair) {
-      var split = pair.split('=');
-      if (split.length === 2) {
-        params[split[0]] = split[1];
-      }
+  queryParams(name) {
+    const parsed = queryString.parse(window.location.search);
+    if (name) {
+      return parsed[name];
+    }
+    return parsed;
+  },
```

Comment:
There were some places where we were using the query-string library, and others where we were using this self-created version (probably taken from some SO answer). Decided to unify around query-string library, which gives us some other abilities.